support/ruby/collins-client/.gitignore

```diff
@@ -5,3 +5,5 @@ rdoc
 .yardoc
 doc
 coverage
+vendor
+.bundle
```

Comment:
We haven't in the past included them.  We could add them, but if we want to then I want to do it in a different PR